app/routes_goals.py

```diff
@@ -0,0 +1,104 @@
+from flask import Blueprint, jsonify, request, abort, make_response
+from .models.task import Task
+from .models.goal import Goal
+from app import db
+from .routes_tasks import complete_or_not, validate_task, get_all_tasks
+
+goal_bp = Blueprint("", __name__, url_prefix="/goals" )
+
+
+# helper function:
+def validate_goal(goal_id):
+    goal = Goal.query.get(goal_id)
+    if not goal:
+        abort(make_response({"message":f"goal {goal_id} not found"}, 404))
+    return goal
+
+# CREATE (CRUD)
+# POST request to /goals
+# request_body = {"title": "My New Goal"}
+@goal_bp.route("", methods=['POST'])
+def create_one_goal():
+    request_body = request.get_json()
+    try:
+        new_goal = Goal(title=request_body['title'])
+    except:
+        return jsonify({"details": "Invalid data"}), 400
```

Comment:
You should avoid having a plain `except` without specifying what you're catching

Also, it's good provide a more specific error message than just "Invalid data", you can always add the exception to your error message if you don't want to add more detail yourself:

```suggestion
    except KeyError as e:
        return jsonify({"details": f"Invalid data: {e}"}), 400
```